### PR TITLE
Update dev preview to include mozillians.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: myke.yml [hunk 1]
```diff
@@ -6,7 +6,6 @@ discover:
   - ../dino-tree
   - ../dino-park-front-end
   - ../iam-profile-faker
-  - ../dino-park-profile-publisher
```
Comment (fiji-flo) [inline myke.yml]: We can also remove `dino-cave` and `iam-profile-faker`

File: git/myke.yml [hunk 2]
```diff
@@ -9,4 +9,5 @@ tasks:
       co.sh dino-park-front-end
       co.sh iam-profile-faker
       co.sh dino-park-profile-publisher
+      co.sh dino-park-search
```
Comment (fiji-flo) [inline git/myke.yml]: can you remove the unneeded repos here, too.

File: external/myke.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 project: external
 desc: external services for DinoPark
 tasks:
-  run-k8s:
+  run-dev:
     cmd: |-
-      kubectl apply -f elastic/k8s/chart
\ No newline at end of file
+      kubectl apply -f elastic/k8s/chart
```
Comment (fiji-flo) [inline external/myke.yml]: (Optional) if you don't mind, could you move `k8s/chart/` → `k8s/tempate` and add a baisc `Chart.yaml`

**Comment by fiji-flo** [comment]
I'd suggest we move the `ingress` for the dev preview from [dino-park-front-end](https://github.com/mozilla-iam/dino-park-front-end/) to this repo.

**Comment by akatsoulas** [comment]
@fiji-flo pr updated

**Pull request merged**